.travis.yml

```diff
@@ -14,6 +14,7 @@ env:
     - VERBOSE: 1
   matrix:
     - PNET_FEATURES="travis" PNET_MACROS_FEATURES="travis"
+    - PNET_FEATURES="travis pcap" PNET_MACROS_FEATURES="travis"
```

Comment:
This effectively doubled the number of builds. Feels very overkill IMO. Could we maybe just run with pcap, and ignore without?

What do you think @mrmonday

Reply:
You're right; we can just build with pcap.